docs/AI-for-security/api/conversation-api-update.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[conversation-api-update]]
+=== Update Conversation
+
+Update an existing Conversation by ID.
+
+==== Request URL
+
+`PUT <kibana host>:<port>/api/security_ai_assistant/current_user/conversations/{id}`
+
+==== Request body
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`title` |String |Conversation title. By default is "New chat". Will be AI generated if not changed. |Yes
+|`apiConfig` |<<update-ApiConfig-obj,apiConfig object>> |Conversation configuration. |Yes
+|`messages` |<<update-message-obj,message object>> |Array of conversation messages. |Yes
+|`replacements` |<<update-replacements-obj,replacements object>> |List of the fields with anonymization. |Yes
+|==============================================
+
+
+[discrete]
+[[update-ApiConfig-obj]]
+== ApiConfig object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`connectorId` |String |Kibana connector ID. |Yes
+|`actionTypeId` |String |Kibana connector action type ID. |Yes
+|`defaultSystemPromptId` |String |Default system prompt ID. |Yes
+|`model` |String |LLM specific model. |Yes
+|==============================================
+
+[discrete]
+[[update-message-obj]]
+== Message object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`role` |String |Message role. Could be "user", "assisatnt" or "system". |Yes
+|`content` |String |Message content to send to LLM. |Yes
+|`isError` |String |Define if the message is an error message and not LLM response. |Yes
+|`timestamp` |String |Timestamp when the message was sent. |Yes
+|`traceData` |Object |Tracing information. |Yes
+|==============================================
+
+[discrete]
+[[update-replacements-obj]]
+== Replacements object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`role` |String |Message role. Could be "user", "assisatnt" or "system". |Yes
+|`content` |String |Message content to send to LLM. |Yes
+|`isError` |String |Define if the message is an error message and not LLM response. |Yes
+|`timestamp` |String |Timestamp when the message was sent. |Yes
+|`traceData` |Object |Tracing information. |Yes
+|==============================================
+
+[discrete]
+=== Example requests
+
+*Example 1*
+
+Update an existing Conversation by ID.
+
+[source,console]
```

Comment:
The Request body section says that the `replacements` object is required, but it's not present in the example.

Reply:
it could be empty.